Assets/Scripts/Enemies/Enemy.cs

```diff
@@ -234,35 +231,34 @@ public void ApplyTimedStatusEffect(StatusEffect statusEffect, Tower originTower)
 public class TimedStatusEffect {
```

Comment:
extract in another file, normally there shouldn't be more than one class in one file